README.md

```diff
@@ -1,2 +1,4 @@
 # code-review-test-repo
 A repo to test code review actions.
```

Comment:
Add second comment as part of an actual review __and__ still as from a collaborator, not author.

Reply:
Resolve conversation, to prepare for another test can be done with a review that requests changes.